### DISCUSSION TITLE
PHP 7.4 support?

### DISCUSSION THREAD

**Discussion by justinbarkhuff** [spatie/laravel-stats#7] [Q&A]
Will this be modified to work with PHP 7.4? Or will this always be PHP 8+?

**Comment by freekmurze** [comment]
No, we don't use PHP 7 anymore. Feel free to fork our package.

**Reply by justinbarkhuff** [reply]
All good. I created a fork which should be backwards compatible with PHP 7.4 and MySQL 5.7. Thanks to you and @AlexVanderbist for all the great work.

https://github.com/justinbarkhuff/laravel-stats
